src/telliot_feeds/reporters/tips/listener/funded_feeds_filter.py

```diff
@@ -183,6 +186,7 @@ async def window_and_priceThreshold_unmet_filter(
 
         Returns: list of feeds that could possibly reward a tip
         """
+        self.prices: dict[bytes, float] = {}
```

Comment:
I think it'd be better / more readable to create the attribute in the constructor.

Reply:
its added here so that it resets every time this function is called and its only needed here and is checked in price_change